lib/helper/helper.go

```diff
@@ -77,10 +77,13 @@ func SaveSession(filename string, config structs.Configuration) error {
 ////////////////////////////////////////////////////////////////////////////////
 
 // PrintSTAK prints out the short term access keys for AWS auth.
-func PrintSTAK(stak kion.STAK, account string) error {
+func PrintSTAK(stak kion.STAK, account string, region string) error {
+	color.Green("Short-term Access Keys for %v:", account)
```

Comment:
We removed this before publishing to github, it looks like you have an old version.  Can you make sure you are starting from github's main branch?